backend/Squirrel.Core/Squirrel.Core.WebAPI/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -49,6 +51,10 @@ public static void AddValidation(this IServiceCollection services)
         services
             .AddControllers()
             .AddFluentValidation(fv => fv.RegisterValidatorsFromAssemblyContaining<NewSampleDtoValidator>());
+
+        services
+            .AddControllers()
```

Comment:
I think we should not duplicate it 

` services.AddControllers()` 

and add class `...DtoValidator` not `ExecutingAssembly`